integration-test/src/config/.env.uat

```diff
@@ -9,6 +9,10 @@ invalid_broker=??
 valid_ci=??
 valid_ci_with_no_station=??
 invalid_ci=??
+valid_pspbroker=??
```

Comment:
??

Reply:
i have forgot to add valid data from UAT environment 🤣